android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/SocialLinkViewHolder.java

```diff
@@ -0,0 +1,113 @@
+package org.fossasia.openevent.adapters.viewholders;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.support.annotation.DrawableRes;
+import android.support.customtabs.CustomTabsIntent;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.extras.SocialLink;
+import org.fossasia.openevent.utils.ConstantStrings;
+import org.fossasia.openevent.utils.Utils;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class SocialLinkViewHolder extends RecyclerView.ViewHolder {
+
+    @BindView(R.id.img_social_link)
+    protected ImageView imageView;
+
+    @BindView(R.id.layout_social_link_parent)
+    protected FrameLayout layout;
+
+    private SocialLink socialLink;
+    private Context context;
+
+    public SocialLinkViewHolder(View itemView, Context context) {
+        super(itemView);
+        ButterKnife.bind(this, itemView);
+
+        this.context = context;
+        imageView.setOnClickListener(view -> {
+            if (socialLink != null && !Utils.isEmpty(socialLink.getLink())) {
```

Comment:
@iamareebjamal I think it won't work because when initializing ViewHolder socialLink will always been null and it will not set onClickListener for any ViewHolder.